pyccl/tests/test_profiles.py

```diff
@@ -548,6 +555,34 @@ def test_hernquist_cumul2d_accuracy(fourier_analytic):
     assert np.all(res2 < 5E-3)
 
 
+def test_einasto_projected_accuracy():
+    cM = ccl.halos.ConcentrationDuffy08(mass_def='200c')
+    # projected profile from quad
+    p1 = ccl.halos.HaloProfileEinasto(mass_def='200c',
+                                      concentration=cM, truncated=False,
+                                      projected_quad=True)
+    # projected profile from FFTLog
+    p2 = ccl.halos.HaloProfileEinasto(mass_def='200c',
+                                      concentration=cM, truncated=False,
+                                      projected_quad=False)
+    # truncated projected profile from FFTLog
+    p3 = ccl.halos.HaloProfileEinasto(mass_def='200c',
+                                      concentration=cM, truncated=True,
+                                      projected_quad=False)
```

Comment:
What's the point of testing against the truncated version now? Should we just remove this bit?

Reply:
The test coverage may drop. Still waiting for the checks to finish.

Reply:
Test coverage dropped. Somehow the `truncated` flag of _real profile is tested here.